run-tests.sh

```diff
@@ -26,16 +27,15 @@ trap finish EXIT
 sphinx-build -qnN docs docs/_build/html
 cookiecutter --no-input -o "$WORKDIR" . \
     project_name=Generated-Fun \
-    database=postgresql \
-    elasticsearch=elasticsearch6
+    database=${COOKIECUTTER_DATABASE:-postgresql} \
+    elasticsearch=${COOKIECUTTER_ELASTICSEARCH:-elasticsearch6}
 
 cd "${WORKDIR}/generated-fun"
 
 git init
 git add -A
 
-pip install -e .\[all\] --quiet
+./scripts/bootstrap
 
 check-manifest -u || true
-
 ./run-tests.sh
```

Comment:
Could we, instead of [manually installing Elasticsearch](https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/43/files#diff-354f30a63fb0907d4ad57269548329e3R45), just do `scripts/setup` and bring all the services needed for Invenio up? It will take good care of spawning the right DB and ES versions plus the rest of the services. Once we need Redis, RabbitMQ... we will have to do the same "manual" installation.

We are doing something similar for OpenData, see [here](https://github.com/cernopendata/opendata.cern.ch/blob/master/.travis.yml#L76), using our docker-compose environment to test.

Reply:
As discussed IRL we do it in a different PR, issue https://github.com/inveniosoftware/cookiecutter-invenio-instance/issues/45.